base/applications/mspaint/selectionmodel.cpp

```diff
@@ -61,17 +61,30 @@ void SelectionModel::DrawBackground(HDC hDCImage, COLORREF crBg)
 void SelectionModel::DrawSelection(HDC hDCImage, COLORREF crBg, BOOL bBgTransparent)
```

Comment:
This one and the other one below, could be made inlined inside the corresponding class' .h file.

Reply:
Done.